tests/models/gpt2/test_modeling_gpt2.py

```diff
@@ -859,3 +868,32 @@ def test_contrastive_search_gpt2(self):
                 "but said in a statement to The Associated Press that"
             ],
         )
+
+    @require_flash_attn
+    @require_torch_gpu
+    @pytest.mark.flash_attn_test
+    @slow
+    def test_flash_attn_2_generate_padding_right(self):
+        """
+        Overwritting the common test as the test is flaky on tiny models
+        """
+        model = GPT2LMHeadModel.from_pretrained("gpt2")
+
+        tokenizer = GPT2Tokenizer.from_pretrained("gpt2")
+
+        texts = ["hi", "Hello this is a very long sentence"]
+
```

Comment:
Let's explicitly write `output_native` 😉

Reply:
Not sure I get this one, the eager attention output is already called `output_native`

Reply:
What I mean is to just add:
```
output_native = ["THis is the expected native output", "And this as well"]` 
```
this helps us in the long run

Reply:
Done here https://github.com/huggingface/transformers/pull/29226/commits/cafc0f12e8906f3a5234007d9b71fea86deae4a1